src/libopensc/pkcs15-syn.c

```diff
@@ -60,9 +61,11 @@ struct sc_pkcs15_emulator_handler builtin_emulators[] = {
 	{ "coolkey",    sc_pkcs15emu_coolkey_init_ex	},
 	{ "din66291",   sc_pkcs15emu_din_66291_init_ex	},
 	{ "esteid2018", sc_pkcs15emu_esteid2018_init_ex	},
+	{ "cardos",     sc_pkcs15emu_cardos_init_ex	},
 
 	{ NULL, NULL }
 };
```

Comment:
clang-format suggested the following change:


```
-	{ NULL, NULL }
-};
+	{NULL, NULL}};
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/pkcs15-syn.c -->